sdk/python/tests/unit/permissions/auth/server/conftest.py

```diff
@@ -1,15 +1,74 @@
 import os
 import tempfile
 from textwrap import dedent
+from typing import Optional, Union
 
 import pytest
 import yaml
 
-from feast import FeatureStore
+from feast import (
+    Entity,
+    FeatureStore,
+    FeatureView,
+    OnDemandFeatureView,
+    StreamFeatureView,
+)
+from feast.permissions.action import AuthzedAction
+from feast.permissions.permission import Permission
+from feast.permissions.policy import RoleBasedPolicy
 from tests.unit.permissions.auth.server.mock_utils import PROJECT_NAME
 from tests.utils.cli_repo_creator import CliRunner
 from tests.utils.http_server import free_port  # noqa: E402
 
+list_permissions_perm = Permission(
+    name="list_permissions_perm",
+    types=Permission,
+    with_subclasses=False,
+    policy=RoleBasedPolicy(roles=["reader"]),
+    actions=[AuthzedAction.READ],
+)
+
+list_entities_perm = Permission(
+    name="list_entities_perm",
+    types=Entity,
```

Comment:
since we are using the same `actions` and `policy`, can we add all the types in a single `Permission` instance, as `types=[Entity, FeatureView, ...]'?

Reply:
It will make the assertion logic more complicated and it is already complicated, I prefer to leave it be if its not that critical